components/merkledb/tests/common/mod.rs

```diff
@@ -19,18 +19,61 @@ use proptest::test_runner::TestCaseResult;
 
 use std::rc::Rc;
 
-use exonum_merkledb::{Database, Fork, TemporaryDB};
+use exonum_crypto::{hash, Hash, HASH_SIZE};
+use exonum_merkledb::{BinaryKey, Database, Fork, ObjectHash, TemporaryDB};
 
 // Max size of the generated sequence of actions.
+//
+// Due external tests running mechanism this file is linked
+// separately with other modules in tests directory. This
+// constant is used in all of then except `proof_map_index`, that's
+// why it marked with `dead_code`.
+#[allow(dead_code)]
```

Comment:
Nit: it make sense to move `Key` to a separate module in order to not force `#[allow(dead_code)]`.